Update AA doc to include default python runbook

jennyhunter-msft opened the pull request:
- Added default python runbook to the list of included resources for RunAs accounts

azurecla commented:
Hi __@brjohnstmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brjohnst). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jennyhunter-msft committed:
Update AA doc to include default python runbook

- Added default python runbook to the list of included resources for RunAs accounts
